README.md

```diff
@@ -1,2 +1,157 @@
+
+6. Edit pg_hba.conf file located in */etc/postgresql/<version>/main/*. Edit the following line:
+    ```
+    local   all             postgres                                <method>
+    ```
+    Change <method> to trust.
+
+7. Start PostgreSQL service with `sudo service postgresql start`. You can also use commands ***restart*** and ***stop*** to control the service.
+
+8. Create database for application by running the following command in Terminal: `createdb -U postgres time_monkey`. If you get ***peer authentication failed*** message, restart PostgreSQL service.
+
+9. Navigate with Terminal to where you want the repository to be cloned and run `git clone https://github.com/Code-Poets/time-monkey.git`
+
+10. Use requirements.lock to install required pacakges to your virtual environment by running `pip install -r requirements.lock`. **Make sure you are using your *virtual* environment first.**
+
+11. Add file to */time_monkey/settings* directory named *local_settings.py* and fill with following content:
+    ```
+    from .development import *
+
+    ALLOWED_HOSTS = ['0.0.0.0', '127.0.0.1', ]      #You can add your machine's IP address here
+
+    SECRET_KEY = 'your secret key goes here'
+    ```       
+    If you have set the password for your PostgreSQL server, make sure to include it in local settings, by writting the following line to the file:
+    ```    
+    DATABASES['default']['PASSWORD'] = <your password>
+    ```
+
+
+### Mac OS
+
+**In order to install required software on your mac, you will need GCC and Homebrew. If you don't have them installed already, refer to the steps below:**
+
+1. GCC can be obtained by downloading XCode, Command Line Tools or OSX-GCC-Installer package. If you already have XCode installed, **DO NOT** install OSX-GCC-Installer. If you perform a fresh install of XCode, you will also need to add the commandline tools by running `xcode-select --install` on the terminal.
+2. To install Homebrew run the following command in Terminal 
+        `ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
+    The script will explain what changes it will make and prompt you before the installation begins. Once you’ve installed Homebrew, insert the Homebrew directory at the top of your PATH environment variable. You can do this by adding the following line at the bottom of your ~/.profile file
+    ```
+    export PATH=/usr/local/bin:/usr/local/sbin:$PATH
+    ```   
+**This project uses Python in version 3.7.0. If you're already using the same or higher version, you can skip the following step.**
+
+3. Run Homebrew installation of Python in Terminal with `brew install python`.
+    You can verify the installation by running `python3 --version`.
+
+**Skip this step if PIP is already installed.**
+
+4. To install PIP run `sudo easy_install pip` in Terminal.
+
+5. If you don't have virtualenv installed, run installation with PIP by typing `pip install virtualenv` in Terminal
```

Comment:
Change it to everything what I have mentioned before in part with installation for Ubuntu about `virtualenv`